### PR TITLE
Mobile compatibility fixes

### PR DESCRIPTION
**Pull request by smaxa20**

### CODE REVIEW

File: components/simple-toast/component.jsx [hunk 1]
```diff
@@ -2,17 +2,21 @@ import React, { PureComponent } from 'react';
...
 /**
  * A mobile first Toast. A toast indicates that an action is being taken that does not require the user's
  * specific attention.
  */
 export class SimpleToast extends PureComponent {
 	static propTypes = {
+		/** Show time on desktop in milliseconds */
+		desktopTime: PropTypes.number,
+		/** Show time on mobile in milliseconds */
+		mobileTime: PropTypes.number,
```
Comment (bryanrsmith) [inline components/simple-toast/component.jsx]: Did we talk about unifying these and having a single duration for both mobile and desktop? I don't really get why we need two (especially with the addition of the dismiss button).

File: components/simple-toast/styled.jsx [hunk 2]
```diff
@@ -115,3 +114,8 @@ export const ToastContainer = styled.div`
 export const ToastContent = styled.div`
 	white-space: nowrap;
 `;
+
+export const ToastClose = styled.div`
```
Comment (bryanrsmith) [inline components/simple-toast/styled.jsx]: We should use a button for this. Generally, interaction handlers should only be attached to interactive/form elements like buttons.

File: components/share-dialog/styled.jsx [hunk 3]
```diff
@@ -17,6 +22,10 @@ export const ShareContainer = styled.div`
 
 export const CopyContainer = styled.div`
 	margin: 0 ${thickness.eight};
+
+	input {
+		text-overflow: ellipsis;
```
Comment (bryanrsmith) [inline components/share-dialog/styled.jsx]: Interesting. Does this work? Usually inputs scroll their contents with caret position.

File: components/simple-toast/component.jsx (hunk 1 above)
Comment (smaxa20) [reply components/simple-toast/component.jsx]: We hadn't talked about it, but I could definitely unify them if that's what we want.

File: components/simple-toast/styled.jsx (hunk 2 above)
Comment (smaxa20) [reply components/simple-toast/styled.jsx]: Ok. I copied this from how it is in modals, should I change it there too?

File: catalog/drop-zone/variations.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ## Component Variations
 
+In general, it is recommended to provide some other alternate UI on mobile .
```
Comment (bryanrsmith) [inline catalog/drop-zone/variations.md]: 👍

**Pull request merged**